### ISSUE TITLE
Sending meta data from form along with webhook

### ISSUE THREAD

**Issue by mrpatg** [agentejo/cockpit#887]
Ok so this is a longshot, but if someone could shed some light it would be great:

I have a form setup in cockpit that I am planning on routing a contact form through. When the form is saved, webhook fires it off to a script which processes the request and sends off an email via mailgun API. The thing is, if I wanted to scale this out to say 10-15 other sites, how could I differentiate the API key in the processing file without having to look them up in a database. I was thinking if I could add some meta data to the form in cockpit, and have that meta data sent along with the webhook that would be awesome. Then I could setup as many forms as I need to, only have a single webhook to handle the form event, and have everything I need to process the message sent back in the webhook post data.

So tl:dr; 
Is it possible to attach meta data to a form "collection" and/or send that back with a form.save webhook?
